### ISSUE TITLE
browser mod device list empty/ browser_id not working

### ISSUE THREAD

**Issue by ronluna** [thomasloven/hass-browser_mod#576]
My Home Assistant version: 0.XX.X
2023.4.4
**What I am doing:**
<img width="645" alt="image" src="https://user-images.githubusercontent.com/9822111/232797967-7e97fcdc-aa82-4d74-98ac-1cc94aa8f489.png">

Browser mod popup is working fine all around. it can even send popup to specific browser ids (browsers are getting registered just fine). Although is unable to propagate the list of registered devices when setting a target in a script. 

**What I expected to happen:**
be able to use browser_id in script so the popup only pops in the browser that triggered the script

**What happened instead:**
No devices can be found
<img width="1047" alt="image" src="https://user-images.githubusercontent.com/9822111/232791121-ba2b1431-ceca-49fd-91af-a2e9000dfb53.png">

**Minimal** steps to reproduce:
Create a basic browser mod popup 

```
service: browser_mod.popup
data:
  dismissable: true
  autoclose: false
  content: test
  browser_id: "{{ browser_id }}"
```

**Error messages from the browser console:**

No error in the console. It even state that the Action was successful but no popup.  Although if a browser_id is manually specified it will pop in that particular browser but the ability to use browser_id in a script does not behave as expected.

<img width="361" alt="image" src="https://user-images.githubusercontent.com/9822111/232793423-4961a5bb-aad1-4367-a744-881e6db38592.png">

---

**By replacing the space in the checkboxes ([ ]) with an X below, I indicate that I:**

- [x] Understand that this is a channel for reporting bugs, not a support forum (https://community.home-assistant.io/).

- [x] Have made sure I am using the latest version of the plugin.

- [x] Have followed the troubleshooting steps of the "Common Problems" section of https://github.com/thomasloven/hass-config/wiki/Lovelace-Plugins.

- [x] Understand that leaving one or more boxes unticked or failure to follow the template above may increase the time required to handle my bug-report, or cause it to be closed without further action.

**Comment by alexisspencer** [comment]
also having the same issue

**Comment by koolzr** [comment]
same here 
but if i don't put any devices it works but it goes to all of the registered browsers

**Comment by alexisspencer** [comment]
even if i put a device ID in it using the "sensor.<id>_browser_path", it goes to all browsers..

**Comment by koolzr** [comment]
> even if i put a device ID in it using the "sensor._browser_path", it goes to all registered browsers..

have you tried rolling back the recent update?

**Comment by alexisspencer** [comment]
no, its the first time i've tried to use browser_mod..  not tried previous versions..

**Comment by davidsnet** [comment]
> also having the same issue

Me as well. I'm thinkin Home Assistant Core 2023.4 update pooched me.

**Comment by kevinbegley** [comment]
Hello,

I am having the same issue. I was wondering if this has been resolved.

**Comment by Perlnx** [comment]
Same issue here.

**Comment by SikaiGGMU** [comment]
same issue

**Comment by sschw56** [comment]
For me as well!

**Comment by thewolfman56** [comment]
I'm now having the same issue.  Old automation would only pop-up on 3 registered entities (using a WebRTC card).  An update in HomeAssistant botched that setup (something between the webRTC and camera stream that I have not had time to nail down), but when I switched the camera to a picture card with a camera live view, the automation would run, but no pop-up.  I disabled the action in the automation, added a new action to pop-up without any target devices and the pop-up works, but unfortunately on every registered device.
